metadata.rb

```diff
@@ -5,7 +5,7 @@
 description      'Installs/Configures mailhog'
 version          '0.1.0'
 
-depends          'runit', '>= 0.0.0'
+depends          'runit', '= 1.7.4'
```

Comment:
The changes made in `1.7.6` have options to disable them. Setting `restart_on_update` to `false` should go back to the previous behavior.